qiskit/transpiler/__init__.py

```diff
@@ -155,6 +229,588 @@
 the ``logical_opt`` pass manager will be called before the ``layout`` stage, and the
 ``scheduling_pm`` pass manager will be used for the ``scheduling`` stage instead of the default.
 
+If you are constructing custom stages for the preset pass managers, you may find some of the
+low-level helper functions in :mod:`qiskit.transpiler.preset_passmanagers` useful.
+
+.. _transpiler-preset-stage-init:
+
+Initialization Stage
+--------------------
+
+.. seealso::
+    `Init stage explanation <https://docs.quantum.ibm.com/guides/transpiler-stages#init-stage>`__
+        Higher-level user-facing explanation of the init stage in the IBM Quantum guide.
+
+The ``init`` stage is responsible for high-level, logical optimizations on abstract circuits, and
+for lowering multi-qubit (3+) operations down to a series of 1- and 2-qubit operations.  As this is
+the first stage run, its input is a fully abstract circuit.  The ``init`` stage must be able to
+handle custom user-defined gates, and all the high-level abstract circuit-description objects, such
+as :class:`.AnnotatedOperation`.
+
+The output of the ``init`` stage is an abstract circuit that contains only 1- and 2-qubit
+operations.
+
+When writing :ref:`stage plugins <transpiler-preset-stage-plugins>`, the entry point for ``init`` is
+``qiskit.transpiler.init``.  The built-in plugins are:
+
+.. list-table::
+    :header-rows: 1
+
+    * - Method
+      - Summary
+
+    * - :ref:`default <transpiler-preset-stage-init-default>`
+      - Built-in unrolling of multi-qubit operations and abstract optimizations.
+
+
+.. _transpiler-preset-stage-init-default:
+
+Built-in ``default`` plugin
+...........................
+
+At optimization level 0, no abstract optimization is done.  The default plugin simply "unrolls"
+operations with more than three qubits by accessing their hierarchical
+:class:`~.circuit.Instruction.definition` fields.
+
+At optimization levels 1 and above, the default plugin also does simple cancellation of adjacent
+inverse gates, such as two back-to-back ``cx`` gates.
+
+At optimization levels 2 and 3, the default plugin enables a much wider range of abstract
+optimizations.  This includes:
+
+* "virtual permutation elision" (see :class:`.ElidePermutations`), where explicit
+  permutation-inducing operations are removed and instead effected as remapping of virtual qubits.
+* analysis of the commutation structure of the IR to find pairs of gates that can be cancelled out.
```

Comment:
```suggestion
* analysis of the commutation structure of the IR to find pairs of gates that can be canceled out.
```
weirdly, we use one L in the US spelling of `canceled`, but two Ls in the spelling of `cancellation`

Reply:
oh man, I don't think I'll ever get my head round some of the doubled/non-doubled consonants in US English - "canceled" looks _so_ wrong to me, like it should be read as "can-sealed" haha